pennylane_lightning_gpu/src/simulator/MPIWorker.hpp

```diff
@@ -81,6 +82,45 @@ inline std::vector<int2> createWirePairs(const int numLocalQubits,
     return wirePairs;
 }
 
+/**
+ * @brief Create wire pairs for bit index swap and transform all target wires to
+ * local ones.
+ *
+ * @param numLocalQubits Number of local qubits.
+ * @param numTotalQubits Number of total qubits.
+ * @param tgts Vector of target wires.
+ * @return wirePairs Wire pairs to be passed to SV bit index swap worker.
+ */
+inline std::vector<int2> createWirePairs(int numLocalQubits, int numTotalQubits,
```

Comment:
Can't this be combined, or just call the implementation with controls?

Reply:
yea, it's possible to call the implementation with controls. This means we need to create an empty std::vector of control where there is no control gate, which make the function call confusing.